lxd/api_internal.go

```diff
@@ -397,6 +397,8 @@ type internalSQLResult struct {
 
 // Perform a database dump.
 func internalSQLGet(d *Daemon, r *http.Request) response.Response {
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
Rather than using `context.Background()` I think it would be better to use `r.Context()` here.
Also, as there's no reason that `cancel` is called until the function returns (and it can't return until `query.Dump()` has returned) then we can probably just pass `r.Context()` directly into `query.Dump()`.

Reply:
Yup... somehow I missed the `r` parameter there.